media/js/firefox/whatsnew/whatsnew-88-en.js

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var jingle = function(){
+        button.style.fontStyle = 'italic';
+        console.log('jingle');
+        var audio = new Audio(button.dataset.audio);
+        audio.play();
+    };
+
+    var button = document.getElementById('outatime');
+
+    button.addEventListener('click', jingle);
+
+    var href = window.location.href;
+
+    var initTrafficCop = function () {
+        if (href.indexOf('v=') !== -1) {
+            if (href.indexOf('v=1') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'wnp-88-control',
+                    'data-ex-name': 'wnp-88-experiment'
+                });
+            } else if (href.indexOf('v=2') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'wnp-88-v2',
+                    'data-ex-name': 'wnp-88-experiment'
+                });
+            } else if (href.indexOf('v=3') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'wnp-88-v3',
+                    'data-ex-name': 'wnp-88-experiment'
+                });
+            }
+            } else if (href.indexOf('v=4') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'wnp-88-v4',
+                    'data-ex-name': 'wnp-88-experiment'
+                });
+            }
+            } else if (href.indexOf('v=5') !== -1) {
```

Comment:
This is throwing a linting error:
` /home/circleci/project/media/js/firefox/whatsnew/whatsnew-88-en.js
  45:15  error  Parsing error: Unexpected token else`

Though it looks OK so I'm not sure what the actual issue is.

Reply:
oh it's the extra `}` and the same on line 38, which closes the previous chain hence why this one is "unexpected" because there was no if to start a new chain.

Reply:
Confirmed. Removing the `}` on lines 38 and 44 fixes the JS.